cargo.sh

```diff
@@ -37,7 +37,14 @@ function print-usage-and-exit {
 [[ $# -gt 0 ]] || print-usage-and-exit
 
 function pkg-meta {
-  cargo metadata --format-version 1 | jq -r ".packages[] | select(.name == \"zerocopy\").$1"
+  # NOTE(#547): We set `CARGO_TARGET_DIR` here because `cargo metadata`
```

Comment:
TODO: Test on the machine which exhibited this behavior to make sure it fixes the problem.

Reply:
The machine on which this problem originally existed seems to have stopped exhibiting the problematic behavior. IMO it's still worth merging this because it's obviously correct, and will help if the problem resurfaces. In the worst case, it just won't do anything.